plugins/modules/packaging/os/homebrew_cask.py

```diff
@@ -540,6 +540,9 @@ def _upgrade_all(self):
             [self.brew_path, 'cask', 'upgrade']
         )
 
+        if self.greedy:
```

Comment:
Please update as well the documentation (`DOCUMENTATION`) of the `greedy` attribute . Right now it mentions only that it is used for outdated command